src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,11 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, String> {
```

Comment:
When you make the changes in Book, remember to change the id of the repository (String) for the new data type (long).
```suggestion
public interface BookRepository extends JpaRepository<Book, Long> {
```

Reply:
In this case, the id should be String type because the ISBN attribute it is the id

Reply:
I didn't check the PR#3 when I see this. Now I understand the change